packages/input-selection/test/LargeFirstSelection/LargeFirstSelection.test.ts

```diff
@@ -0,0 +1,162 @@
+import { Cardano } from '@cardano-sdk/core';
+import { LargeFirstSelector } from '../../src';
+import { MOCK_NO_CONSTRAINTS, mockConstraintsToConstraints } from '../util/selectionConstraints';
+import {
+  MockChangeAddressResolver,
+  asAssetId,
+  asTokenMap,
+  assertInputSelectionProperties,
+  getCoinValueForAddress,
+  mockChangeAddress
+} from '../util';
+import { TxTestUtil } from '@cardano-sdk/util-dev';
+
+describe('LargeFirstSelection', () => {
+  it('consumes only the largest ADA UTXOs needed and leaves the remainder', async () => {
```

Comment:
all utxos in the test have the same value (`3_000_000n`) so the test name is not accurate for what's being asserted
```suggestion
  it('consumes just enough ADA UTXOs needed and leaves the remainder', async () => {
```

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1621/commits/c64fabc330187e9a108841491ba761d093df096f

Reply:
also added a few more tests to better assert the large first behaviour https://github.com/input-output-hk/cardano-js-sdk/pull/1621/commits/4f3f9f261d409e1b5c3185e86c3a1c5f641328ec